src/Generators/AbstractTestsGenerator.php

```diff
@@ -186,7 +187,6 @@ protected function getMockModel($model): array
     protected function generateFixtures(): void
     {
         $object = $this->getFixtureValuesList($this->model);
-        $entity = Str::snake($this->model);
```

Comment:
let's implement abstract method `getTestingEntityName` and get entity name from it instead of from the magic class property.